src/components/autocomplete/js/autocompleteController.js

```diff
@@ -106,7 +106,12 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         left   = hrect.left - root.left,
         width  = hrect.width,
         offset = getVerticalOffset(),
+        position = $attrs.mdDropdownPosition,
         styles;
+    // Automatically determine dropdown placement based on available space in viewport.
```

Comment:
While you are at it, can you add an empty line before the comment (just looks better then)